pages/api/reservoir/[...slug].ts

```diff
@@ -33,6 +34,25 @@ const proxy = async (req: NextApiRequest, res: NextApiResponse) => {
   setParams(url, query)
 
   if (endpoint.includes('redirect/')) {
+    // Redirect eth and weth currency icons to self-hosted
+    // versions without any padding
+    endpoint = endpoint.toLowerCase()
+
+    if (
+      (chainPrefix == 'ethereum' || chainPrefix == 'goerli') &&
+      endpoint.includes('currency')
+    ) {
+      if (endpoint.includes(constants.AddressZero)) {
+        res.redirect('/icons/currency/new-eth.png')
+        return
+      } else if (
+        endpoint.includes('0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2') ||
```

Comment:
Is there any way we can use the map that we have for wrappedContracts in the utils? Just to avoid hardcoding it in too many places.